docs/docs/Concepts/mcp-server.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Model Context Protocol (MCP) server
+slug: /mcp-server
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Icon from "@site/src/components/icon";
+
+Langflow integrates with the [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) as both an MCP server and an MCP client.
+This page describes how to use Langflow as an *MCP server*.
+For information about using Langflow as an *MCP client*, see the [MCP component](/components-tools#mcp-server).
+
+As an MCP server, Langflow exposes your flows as [tools](https://modelcontextprotocol.io/docs/concepts/tools) that [MCP clients](https://modelcontextprotocol.io/clients) can use use to take actions.
+
+## Prerequisites
+
+* A Langflow project with at least one flow created.
+
+* To use MCP Inspector, any LTS version of [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed on your computer.
```

Comment:
```suggestion
* Any LTS version of [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed on your computer to use MCP Inspector to [test and debug flows](#test-and-debug-flows).
```

Reply:
Note that this change depends on a new anchor that's added in another suggestion.